src/all/deviantart/src/eu/kanade/tachiyomi/extension/all/deviantart/DeviantArt.kt

```diff
@@ -0,0 +1,163 @@
+package eu.kanade.tachiyomi.extension.all.deviantart
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.jsoup.parser.Parser
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class DeviantArt : HttpSource() {
+    override val name = "DeviantArt"
+    override val baseUrl = "https://deviantart.com"
+    override val lang = "all"
+    override val supportsLatest = false
+
+    private val backendBaseUrl = "https://backend.deviantart.com"
+    private fun backendBuilder() = backendBaseUrl.toHttpUrl().newBuilder()
+
+    private val dateFormat by lazy {
+        SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz", Locale.ENGLISH)
+    }
+
+    private fun parseDate(dateStr: String): Long {
+        return try {
+            dateFormat.parse(dateStr)!!.time
+        } catch (_: ParseException) {
+            0L
+        }
+    }
+
+    override fun popularMangaRequest(page: Int): Request {
+        throw UnsupportedOperationException(SEARCH_ERROR_MSG)
+    }
+
+    override fun popularMangaParse(response: Response): MangasPage {
+        throw UnsupportedOperationException(SEARCH_ERROR_MSG)
+    }
```

Comment:
The error message assumes the user is using an incorrect query. Should do one of:
- Default to some well-known gallery
- Reword telling the user how to use it
-

Reply:
Is there a way to display a message without having it start with `UnsupportedOperationException`?